components/CanvasOne.js

```diff
@@ -0,0 +1,82 @@
+import { useRef, useEffect, useState } from "react";
+import styled from "styled-components";
+
+function lineWidthRandomizer() {
+  const randomNumber = Math.random();
+  if (randomNumber <= 0.4) {
+    return 2;
+  } else if (randomNumber <= 0.8) {
+    return 8;
+  } else {
+    return 4;
+  }
+}
+
+let firstLineWidth = lineWidthRandomizer();
+let secondLineWidth = lineWidthRandomizer();
+let thirdLineWidth = lineWidthRandomizer();
+let fourthLineWidth = lineWidthRandomizer();
+
+export default function CanvasOne() {
+  const canvasRef = useRef(null);
+  const [backgroundColor, setBackgroundColor] = useState(148);
+  useEffect(() => {
+    const canvas = canvasRef.current;
+    const context = canvas.getContext("2d");
+    // Beginn drawing statements
+    context.fillStyle = `hsl(${backgroundColor}, 60%, 95%)`;
+    context.fillRect(0, 0, canvas.width, canvas.height);
+
+    context.fillStyle = "hsl(148, 100%, 50%)";
```

Comment:
don't you want to use the backgroundColor here as well? (its the same hue value)

Reply:
For now not, because in this user story I only want to change the background color. 

In the next user story i will change that slider into a general color changer.